Sources/MapboxMaps/MapView/MapView.swift

```diff
@@ -29,8 +29,14 @@ open class MapView: BaseMapView {
     /// A reference to the `EventsManager` used for dispatching telemetry.
     internal var eventsListener: EventsListener!
 
-    public init(frame: CGRect, resourceOptions: ResourceOptions, glyphsRasterizationOptions: GlyphsRasterizationOptions = GlyphsRasterizationOptions.default, styleURI: StyleURI? = .streets) {
-        super.init(frame: frame, resourceOptions: resourceOptions, glyphsRasterizationOptions: glyphsRasterizationOptions, styleURI: styleURI?.rawValue)
+    /// Initialize a MapView
+    /// - Parameters:
+    ///   - frame: frame for the MapView.
+    ///   - mapInitOptions: `MapInitOptions`; default uses `CredentialsManager.default`
+    ///         to retrieve a shared default access token.
+    ///   - styleURI: `StyleURI` to initialize the map with, or nil.
+    public init(frame: CGRect, mapInitOptions: MapInitOptions = MapInitOptions(), styleURI: StyleURI? = .streets) {
+        super.init(frame: frame, mapInitOptions: mapInitOptions, styleURI: styleURI?.rawValue)
```

Comment:
Looks like BaseMapView still takes a URL? Should we align them?

Reply:
Let's leave for the bigger discussion about the style uri type. This is likely to be removed from the initializers anyway.